app/src/test/java/com/example/mpvcalculatorrx/CalculatorPresenterMockModelUnitTest.java

```diff
@@ -0,0 +1,189 @@
+package com.example.mpvcalculatorrx;
+
+
+import com.example.mpvcalculatorrx.mvp.model.CalculatorModel;
+import com.example.mpvcalculatorrx.mvp.presenter.CalculatorPresenter;
+import com.example.mpvcalculatorrx.mvp.view.CalculatorView;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static com.example.mpvcalculatorrx.CalculatorPresenterRealModelUnitTest.getExpectedResult;
+import static com.example.mpvcalculatorrx.CalculatorPresenterRealModelUnitTest.verifyDisplayUpdatesForOperation;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class CalculatorPresenterMockModelUnitTest {
+
+    @Mock
+    CalculatorModel mockModel;
+    @Mock
+    CalculatorView mockView;
+    private CalculatorPresenter presenter;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+
+        presenter = new CalculatorPresenter(mockModel, mockView);
+
+        assertNotNull(mockModel);
+        assertNotNull(mockView);
+        assertNotNull(presenter);
+    }
+
+    @Test
+    public void shouldAddDigit() {
+        presenter.onDigitPressed(1);
+        verify(mockModel).addDigit(1);
+        verify(mockView).setDisplayValue(anyString());
```

Comment:
Improve your test, don't expect to see something, assert that it is displaying that it should be.